Recipely/Recipely/NetworkService/RecipeDetailResponseDTO.swift

```diff
@@ -0,0 +1,80 @@
+// RecipeDetailResponseDTO.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// RecipeDetailResponseDTO
+struct RecipeDetailResponseDTO: Codable {
+    /// Массив с подробными рецептами
+    let hits: [RecipeDetailHitDTO]
+}
+
+/// RecipeDetailHitDTO
```

Comment:
```suggestion
/// RecipeDetailHitDTO
```
дока не соответствует действительности